compute_tools/src/installed_extensions.rs

```diff
@@ -38,61 +37,77 @@ fn list_dbs(client: &mut Client) -> Result<Vec<String>> {
 /// Connect to every database (see list_dbs above) and get the list of installed extensions.
 ///
 /// Same extension can be installed in multiple databases with different versions,
-/// we only keep the highest and lowest version across all databases.
+/// so we report a separate metric (number of databases where it is installed)
+/// for each extension version.
 pub fn get_installed_extensions(mut conf: postgres::config::Config) -> Result<InstalledExtensions> {
     conf.application_name("compute_ctl:get_installed_extensions");
     let mut client = conf.connect(NoTls)?;
-
     let databases: Vec<String> = list_dbs(&mut client)?;
 
-    let mut extensions_map: HashMap<String, InstalledExtension> = HashMap::new();
+    let mut extensions_map: HashMap<(String, String, String), InstalledExtension> = HashMap::new();
     for db in databases.iter() {
         conf.dbname(db);
         let mut db_client = conf.connect(NoTls)?;
-        let extensions: Vec<(String, String)> = db_client
+        let extensions: Vec<(String, String, i32)> = db_client
             .query(
-                "SELECT extname, extversion FROM pg_catalog.pg_extension;",
+                "SELECT extname, extversion, extowner::integer FROM pg_catalog.pg_extension",
```

Comment:
Just in case, you can do `extowner::regrole::text` to get a role name (I'd only double-check that it's also the case for all our PG versions)

Then we can check by just `cloud_admin`. Otherwise, hardcoded oid looks a bit odd to me. I guess it should be stable because that's the superuser created by `initdb`, but still

What do you think?

Reply:
10 is the oid of the superuser created by initdb and it is stable.
I used it to not depend on the superuser name. 

Also, the query I wrote will only use one catalog table (extowner->text will implicitly query another catalog). Performance is not  a big deal here, but still.

I can add extowner name for sanity, if you think this will be better.